TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/configure.py

```diff
@@ -107,6 +108,51 @@ def _parse_config_from_stdin(stdin_content: str) -> WorkflowConfig:
         raise click.ClickException(f"Failed to parse input as YAML: {e}")
 
 
+def _get_basic_creds_secrets_in_config(secret_store: SecretStore, new_config: WorkflowConfig):
+    """Scrape any auth credentials secrets that need to be created."""
+
+    from ..services.script_runner import ScriptRunner
+    runner = ScriptRunner()
+    result = runner.get_basic_creds_secrets_in_config(new_config.to_yaml())
+    logger.info(f"got back script result for get_secrets_in_config: {result}")
+
+    if 'invalidSecrets' in result and (invalid_secrets := result['invalidSecrets']):
+        raise click.ClickException(f"Invalidly named secrets found: {invalid_secrets}")
+    elif 'validSecrets' in result and (valid_secrets := result.get('validSecrets', [])):
+        valid_existing_secrets = list(filter(secret_store.secret_exists, valid_secrets))
+        return set(valid_existing_secrets), list(set(valid_secrets) - set(valid_existing_secrets))
+    else:
+        return [], []
+
+
+def _handle_add_basic_creds_secrets(secret_store, missing_secrets):
+    click.echo("Some secrets don't yet exist.  "
```

Comment:
maybe "Some secrets used in the workflow cluster definitions" or "in the workflow configuration"